shinydashboard/lantern/server.R

```diff
@@ -134,8 +134,29 @@ function(input, output, session) { #nolint
      "validations_tab" = "Validations Page"
   )
 
+  output$resource_tab_popup <- renderUI({
+    if (show_resource_tab_popup()) {
+      actionButton("resource_popup", "How to use this page")
+    }
+  })
+
+  observeEvent(input$resource_popup, {
+    showModal(modalDialog(
+      title = "How to use this page...",
+      p("By default, the list of resources below contains the supported resources across all endpoints and FHIR versions. Clicking a resource in the left box selects it and moves it to the right box. Remove a resource from the list by clicking the resource in the right box.", style = "font-size:16px; margin-left:5px;"),
+      p("You may also change the FHIR Version or Developer filtering criteria to select the applicable supported resources from the default list.
+        Any selected resources at that point will be removed if no endpoints that pass the selected filtering criteria support the given resource.
+        Resources that are filtered out of the selected list will not re-appear in the list if you make other changes to the FHIR Version or Developer filtering criteria.", style = "font-size:16px; margin-left:5px;"),
+      p("You must either", style = "font-size:16px; margin-left:5px;"),
+      p("1. Select a resource from the left side of the resources box to select it and add it to the list, which moves it to the right side of the box", style = "font-size:16px; margin-left:5px;"),
+      p("or", style = "font-size:15px; margin-left:5px;"),
+      p("2. Click the 'Select All Resources' button to add all resources that are supported by the endpoints passing the selected criteria.", style = "font-size:16px; margin-left:5px;"),
```

Comment:
I think we can get rid of these 4 lines and change it to "You will have to re-select these resources, either by clicking the resource on the left box, or clicking the 'Select All Resources' button."